app/services/force/base.rb

```diff
@@ -41,9 +41,29 @@ def debug(q)
     end
     # run a Salesforce SOQL query
 
+    # This method was added to help developers debug/code faster.
+    # because some queries take way too long (like Applications)
+    # It's not designed to be used in prod.
+    # it can be enable by setting CACHE_SALESFORCE_REQUESTS=yes
+    # I'm not using cache_query since the logic it's different. cache_query should be removed. Also, it's not being used.
+    # Fed
+    def query_with_cache(q)
```

Comment:
@chrisdolendo @callaghanc @creativecoder I added this crazy method to cache responses to improve coding productivity. Some queries take too long and moving around the app it's kind of slow. We talked about caching, but this is `caching for developers`. Let me know if we can keep it or if we should remove it. There is an extra check in `query` to make this work. https://github.com/Exygy/sf-dahlia-lap/pull/26/files#diff-b6d0b13d1138c63c05d714686a5426b0R62

Reply:
👍